_posts/2024-09-15-labels-talk/labels-talk.Rmd

```diff
@@ -0,0 +1,158 @@
+---
+title: "Iterating to achieve my first posit::conf talk"
+description: |
+  Three submissions regarding labelled data, two rejections and one success
+author:
+  - name: Shannon Pileggi
+    url: {}
+date: 2024-09-15
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+## TL;DR
+
+This year I was thrilled to speak at posit::conf for the first time ever on [Context 
+is King](https://www.pipinghotdata.com/talks/2024-08-13-context-is-king/), a talk about variable labels in R. Part of my elation was due to actually 
+having a talk accepted after two prior rejections on related submissions.
+
+## Overview
+
+I have attended six out the eight past RStudio / Posit conferences, and I have
+instructed a workshop at the past three. Up until 2024, I had actually never 
```

Comment:
links to the workshop resources?

also I recommend line breaks after each sentence as it makes reviews easier :wink: